Clean-up/test_clean_lambda.py

```diff
@@ -0,0 +1,54 @@
+"""This file tests whether the clean_lambda file functions correctly"""
+
+import json
+from unittest.mock import patch, MagicMock
+
+import pytest
+
+from clean_lambda import delete_unsubscribed, handler
+
+
+@pytest.fixture(name="unsubscribed_products")
+def fixture_unsubscribed_products() -> list[dict]:
+    """Example data for unsubscribed products"""
+    return [{"product_id": 4, "url": "http://example.com/product4",
+             "product_code": "P004", "product_name": "Product 4"},
+            {"product_id": 5, "url": "http://example.com/product5",
+             "product_code": "P005", "product_name": "Product 5"}]
```

Comment:
Consider moving this into a `conftest.py` file.

Reply:
Done